test/lint/lint-circular-dependencies.sh

```diff
@@ -13,7 +13,6 @@ EXPECTED_CIRCULAR_DEPENDENCIES=(
     "index/txindex -> validation -> index/txindex"
     "policy/fees -> txmempool -> policy/fees"
     "qt/addresstablemodel -> qt/walletmodel -> qt/addresstablemodel"
-    "qt/bantablemodel -> qt/clientmodel -> qt/bantablemodel"
     "qt/bitcoingui -> qt/utilitydialog -> qt/bitcoingui"
```

Comment:
Was this meant to delete the `"qt/bitcoingui -> qt/utilitydialog -> qt/bitcoingui"` circular dependency instead? [Travis](https://travis-ci.org/bitcoin/bitcoin/jobs/644136155#L371) thinks it has been removed.

Reply:
Ops, yes.. this was done on top of the another PR and on rebased I've picked the wrong line.

Reply:
Fixed.